use-cases/inferencing/cost-optimization/gcsfuse/README.md

```diff
@@ -0,0 +1,271 @@
+  | ------------- | ------------------------------------------------------------------------------ | ---------------------- |
+  | ACCELERATOR   | Type of GPU accelerator to use (a100, h100, l4)                                | a100                   |
+  | MODEL_NAME    | The name of the model folder in the root of the GCS model bucket               | meta-llama             |
+  | MODEL_VERSION | The name of the version folder inside the model folder of the GCS model bucket | Llama-3.3-70B-Instruct |
+
+  ```sh
+  ACCELERATOR="a100"
+  MODEL_NAME="meta-llama"
+  MODEL_VERSION="Llama-3.3-70B-Instruct"
+  ```
+
+## Serve the model with vLLM with no tuning
+
+- Configure the deployment.
+
+  ```
+  VLLM_IMAGE_NAME="vllm/vllm-openai:v0.6.6.post1"
+  ```
+
+  ```sh
+  git restore manifests/model-deployment-${ACCELERATOR}-dws.yaml
+  sed \
+  -i -e "s|V_MODEL_BUCKET|${MLP_STORAGE_BENCHMARK_FLAT_BUCKET}|" \
+  -i -e "s|V_MODEL_NAME|${MODEL_NAME}|" \
+  -i -e "s|V_MODEL_VERSION|${MODEL_VERSION}|" \
+  -i -e "s|V_IMAGE_NAME|${VLLM_IMAGE_NAME}|" \
+  -i -e "s|V_KSA|${MLP_STORAGE_BENCHMARKING_KSA}|" \
+  manifests/model-deployment-${ACCELERATOR}-dws.yaml
+  ```
+
+- Create the provisioning request and the deployment.
+
+  ```sh
+  kubectl --namespace ${MLP_KUBERNETES_NAMESPACE} apply -f manifests/provisioning-request-${ACCELERATOR}.yaml
+  kubectl --namespace ${MLP_KUBERNETES_NAMESPACE} apply -f manifests/model-deployment-${ACCELERATOR}-dws.yaml
+  ```
+
+  You will see the output similar to the following:
+
+  ```
+  podtemplate/a100-storage-benchmark created
+  provisioningrequest.autoscaling.x-k8s.io/a100-storage-benchmark created
+  deployment.apps/vllm-openai-gcs-llama33-70b-a100 created
+  service/vllm-openai-gcs-llama33-70b-a100 created
+  ```
+
+  > Note : It may take a few minutes before the provisioning request is accepted
+  > and the resources are provisioned. The deployment will be started as soon as
+  > the resources are provisioned.
+
+- Check the status of the provisioning request, once the `PROVISIONED` column
+  shows `True`, the deployment will start.
+
+  ```sh
+  watch -n 5 kubectl --namespace ${MLP_KUBERNETES_NAMESPACE} get provisioningrequest a100-storage-benchmark
+  ```
+
+- When the deployment has started, watch the it until it is ready and available.
+
+  ```sh
+  watch --color --interval 5 --no-title \
+  "kubectl --namespace ${MLP_MODEL_SERVE_NAMESPACE} get deployment/vllm-openai-gcs-llama33-70b-${ACCELERATOR} | GREP_COLORS='mt=01;92' egrep --color=always -e '^' -e '1/1     1            1'"
+  ```
+
+  It can take 20+ minutes for the deployment to be ready and available.
+
+  ```
+  NAME                              READY   UP-TO-DATE   AVAILABLE    AGE
+  vllm-openai-gcs-llama33-70b-a100   1/1     1            1           XXXXX
+  ```
+
+## Calculate pod startup time
+
+- Install bc in CloudShell
+
+  ```sh
+  sudo apt-get install bc
```

Comment:
Are you using `bc` so that you can do floating point arithmetic?

Reply:
removed bc to do direct expressions since I am already converting all time differences to milliseconds